tests/virt_autotest/switch_to_ssh_and_install_hypervisor.pm

```diff
@@ -50,9 +50,21 @@ sub run {
     zypper_call('--gpg-auto-import-keys ref');
     zypper_call("in -t pattern ${hypervisor}_server ${hypervisor}_tools", 1800);
     set_serial_console_on_vh('', '', $hypervisor);
+    systemctl 'enable libvirtd', ignore_failure => 1;
 
     virt_autotest::utils::install_default_packages();
     save_screenshot;
+
+    #set up necessary network bridge br0 for virtualization tests.
```

Comment:
This file goes to contain more and more minor functions that can not be indicated from the file name. May need to consider changing the file name at some time.

Reply:
br0 is still one of the necessary part of installing hypervisor. it is created by default in SLE when kvm/xen role is chosen during installation. while in TW, you have to zypper in kvm/xen, then set up br0 manually. the procedure is still conforms to the file name yet.